CPS-0010/README.md

```diff
@@ -0,0 +1,276 @@
+Due to the nature of these connections, there can often be unforeseen impacts from small design decisions.
+
+For users, connectors should offer secure and reliable compatibility with a wide range of dApps.
+For dApps, connectors should be reliable and provide stable and rich APIs.
+For wallets, connectors should be secure and extendable, and you should not expect wallets to go beyond their standard activities.
+
+### Core Concerns
+
+#### 1. Secure connection
+Security should remain of paramount concern within Web3.
+Connection standards must strive to, above all, not compromise the security of wallets or dApps.
+Connection standards should be aware of the potential impacts of standard security vulnerabilities, such as man-in-the-middle attacks.
+
+#### 2. Secure API
+The security of the API itself should again remain eminent.
+This means that no secret information should ever be allowed to leave the wallet.
+
+#### 3. Range of supported connection
+Connection standards should support a wide range of wallets and dApp platforms.
+This means we shouldn't assume a software environment (e.g., JavaScript in the browser) and define the APIs using schema languages widely used in language-agnostic contexts.
+With the base-level connections only offering an indication of supported APIs bidirectionally.
+
+#### 4. Expressive API
+APIs should allow for an expressive range of information to be exchanged.
+Communication should be expressive in both directions.
+APIs should be language agnostic.
+
+#### 5. Versioned
+Connection standards and APIs should be explicitly versioned to clearly allow upgrades.
+Furthermore, ideally, APIs should allow for optional extendability to facilitate specialization of connection.
+
+#### 6. Respect the role of wallet
+APIs should be written to have a clear scope, understanding the potential strains placed on wallet providers.
+APIs, at a base level, should focus on preserving the base [role of the wallet](#clear-role-of-wallet).
+
+### Context
+
+#### Cardano Connector History
+[CIP-30](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0030/README.md) is *the* fundamental wallet connector standard for Cardano.
+This standard has facilitated the emergence of dApp development on Cardano by defining both a connection standard and an API.
+It is, to date, the only wallet connector standard to see wide adoption in the ecosystem.
+Using an injected Javascript object for communication between wallets and web-based stacks
+Since its' authoring, CIP-30 has seen continued iteration with changes to its API.
+
+##### CIP-30 Alternatives
+Since CIP-30's authoring there have only been two other competing standards, in CIP-45 | Decentralized WebRTC dApp-Wallet Communication ([#395](https://github.com/cardano-foundation/CIPs/pull/395)) and CIP-90 | Extendable dApp-Wallet Web Bridge ([#462](https://github.com/cardano-foundation/CIPs/pull/462/)).
+Although neither of of this standards have seen wide adoption.
```

Comment:
```suggestion
Neither of these standards have seen wide adoption.
```
(grammar) Another way to fix the hanging clause (a sentence fragment) is to run the 2 sentences together with a comma before "Although".